setup.py

```diff
@@ -13,11 +13,47 @@
 # governing permissions and limitations under the License.
 
 from setuptools import setup, find_packages
+from os.path import dirname, join
+import os, sys
+from os.path import exists
+readme_file = 'README.rst'
+
+def _get_readme():
+    with open(join(dirname(__file__), readme_file)) as f:
+        return f.read()
+
+hcfg = "/etc" if os.access('/etc/', os.W_OK) else ''
+daemon_dir = '/usr/lib/systemd/system' if exists('/usr/lib/systemd/system') \
+		and os.access('/usr/lib/systemd/system', os.W_OK)\
+		 else ('/lib/systemd/system' if os.access('/lib/systemd/system', os.W_OK) else '')
 
 setup(name='haas',
       version='0.2rc2',
+      maintainer= 'Developers of the HaaS Project at MOC',
+	author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
       url='https://github.com/CCI-MOC/haas',
+      description='A bare-metal isolation service that automates allocation and management ' \
+		  'of non-virtualized compute resources across mutually untrusting ' \
+		  'and incompatible provisioning systems.',
+      long_description=_get_readme(),
+      license='Apache 2.0',
+      classifiers=['Development Status :: 4 - Beta',
+			'Intended Audience :: Developers',
```

Comment:
Just downloaded it to have a look. You've got literal tab characters in the file. Please replace those with space. In general, *never* put tab characters in python source files. I've seen a few other files that have tabs in them, please configure your editor so this doesn't happen again.